src/models/stores/groups.ts

```diff
@@ -92,6 +97,9 @@ export const GroupsModel = types
       userInGroup(uid: string, groupId?: string) {
         const groupUser = self.groupForUser(uid);
         return !!(groupId && groupUser && (groupUser.id === groupId));
+      },
+      documentForGroup(groupId: string) {
```

Comment:
```suggestion
      virtualDocumentForGroup(groupId: string) {
```
Suggesting it for consistency with the suggested change above.